engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/PreviewWorldScreen.java

```diff
@@ -224,18 +193,19 @@ private void updatePreview() {
         final WaitPopup<ByteBufferResult> popup = manager.pushScreen(WaitPopup.ASSET_URI, WaitPopup.class);
         popup.setMessage("Updating Preview", "Please wait ...");
 
-        final ByteBufferProgressListener progressListener = new ByteBufferProgressListener() {
-            @Override
-            public void onProgress(float progress) {
+        ProgressListener progressListener = progress ->
                 popup.setMessage("Updating Preview", String.format("Please wait ... %d%%", (int) (progress * 100f)));
-            }
-        };
+
+        final int imageSize = 384;
```

Comment:
Magic number? Maybe a comment or declaration up top? I see it used to be near class top